api/specs.proto

```diff
@@ -409,8 +409,15 @@ message SecretSpec {
 	// - golang: Go templating
 	Driver templating = 3;
 
-	// Driver is the the secret driver that is used to store the specified secret
+	// Driver is the the secret driver that is used to store the specified secret.  One example
```

Comment:
Wondering if using the driver here for external CA makes sense, or if we should just use the same mechanism as the root CA (e.g. the `ExternalCA` object)